### DISCUSSION TITLE
case-studies/indias-space-odyssey/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#191] [Website Discussions]
# India’s Space Odyssey

India’s Space Odyssey, the science documentary produced by Miditech Studios for Warner Bros. Discovery/Discovery+ India released in October, 2021.

https://fcp.cafe/case-studies/indias-space-odyssey/

**Comment by latenitefilms** [comment]
This is such a great read! Thanks so much Palok Singh!
